components/concept/storage/build.gradle

```diff
@@ -22,19 +22,10 @@ android {
 }
 
 dependencies {
-    implementation Dependencies.kotlin_stdlib
-    implementation Dependencies.support_annotations
-    implementation Dependencies.kotlin_coroutines
-
-    // We expose this as API because we are using Observable in our public API and do not want every
-    // consumer to have to manually import "base".
-    api project(':support-base')
-
-    testImplementation Dependencies.testing_junit
-    testImplementation Dependencies.testing_robolectric
-    testImplementation Dependencies.testing_mockito
-
-    testImplementation project(':support-test')
+    // Necessary because we use 'suspend'. Fun fact: this module will compile just fine without this
+    // dependency, but it will crash at runtime.
+    // Included via 'api' because this module is unusable without coroutines.
+    api Dependencies.kotlin_coroutines
```

Comment:
This one I do not really understand. `suspend` is a language feature and shouldn't require the coroutines library. The library code itself also doesn't reference any code from the package. What kind of crash are you seeing?

Reply:
`Continuation` will be missing from the classpath.